tests/dummy/app/routes/nodes/detail/draft-registrations/index.js

```diff
@@ -8,6 +8,8 @@ export default Ember.Route.extend({
     },
     setupController(controller, model) {
         this._super(controller, model);
+        var metaschemas = this.store.findAll('metaschema');
+        controller.set('metaschemas', metaschemas);
```

Comment:
Local is working fine for me - I have all the metaschemas loaded when I navigate to the draft index view.  Making sure the promise has resolved might be important however.  Do you know how I would do this within the controller?  Alternatively, is there another way to have the metaschemas available besides setting them on the controller?   Even though this is a draft route, I still need metaschema.  I used to return a RSVP.hash and return two models, but this is unreliable. 

As an aside, that API response you were getting above is exactly what I was getting on staging - for that and several other types of requests.  Not seeing this locally, however.